src/actor/Claimer.spec.ts

```diff
@@ -302,6 +307,33 @@ describe('Claimer', () => {
     )
     expect(Array.isArray(presentation.body.content))
   })
+  it('create public presentation from request without peRequest', async () => {
+    const body: IRequestClaimsForCTypes = {
+      type: MessageBodyType.REQUEST_CLAIMS_FOR_CTYPES,
+      content: {
+        allowPE: false,
+        ctypes: ['this is a ctype hash'],
+      },
+    }
+    const request = new Message(body, verifier, claimer.getPublicIdentity())
+
+    const presentation = await Claimer.createPresentation(
+      claimer,
+      request,
+      verifier.getPublicIdentity(),
+      [credentialPE],
+      [attester.getPublicIdentity()],
+      false
+    )
+    expect(presentation.body.type).toEqual(
+      MessageBodyType.SUBMIT_CLAIMS_FOR_CTYPES_PUBLIC
+    )
+    expect(Array.isArray(presentation.body.content))
```

Comment:
Missing `.toBeTruthy()` or `.toBe(true)` here. 

```suggestion
    expect(Array.isArray(presentation.body.content)).toBe(true)
```

Reply:
fixed in ea38166a4ce1b9c442db0264098e7f7ff080847f